testing/functional_tests_assertions.rst

```diff
@@ -0,0 +1,83 @@
+.. index::
+   single: Tests; Assertions
+
+Functional Test specific Assertions
+===================================
+
+.. versionadded:: 4.3
+
+    The shortcut methods for assertions using ``WebTestCase`` were introduced
+    in Symfony 4.3.
+
+When doing functional tests, sometimes you need to make complex assertions in
+order to check whether the ``Request``, the ``Response`` or the ``Crawler``
+contain the expected information to make your test succeed.
+
+Here is an example with plain PHPUnit::
+
+    $this->assertGreaterThan(
+        0,
+        $crawler->filter('html:contains("Hello World")')->count()
+    );
+
+Now here is the example with the assertions specific to Symfony::
+
+    $this->assertSelectorTextContains('html', 'Hello World');
+
+.. note::
+
+    These assertions only work if a request has been made with the ``Client``
+    in a test case extending the ``WebTestCase`` class.
+
+Assertions Reference
+---------------------
+
+Response
+~~~~~~~~
+
+- ``assertResponseIsSuccessful()``
```

Comment:
Maybe it is usefull to have the full signature here or transform them all to ```:method:`` ``` links?

Reply:
We decided to not do this because Symfony API isn’t available anymore.

Yesterday I talked to @Nyholm, @dmaicher and @xabbuh about fixing this by unsung github. MaBe you can talk to each other directly

Reply:
We cannot link to a specific method when using GitHub, so that's a bit sad. For class, it will be a good solution though. We must discuss whether it's usefull to link functions, or just like to the class on github directly

Reply:
in a PR github seems to have this method information. Otherwiese we could have a proxy wich calculates the needed line number on demand  for the github url